src/modules/submissions.js

```diff
@@ -2228,7 +2228,7 @@
 					// adds the appropriate birth/death year categories
 					newText.append( '\n{{subst:L' +
 						'|1=' + data.birthYear +
-						'|2=' + ( data.deathYear || '' ) +
+						'|2=' + ( data.deathYear || 'MISSING' ) +
```

Comment:
If the person picks "living" in the combo box, this should probably be ''. If the person picks "dead", it should be "MISSING". If the person provides a year, it should be the year.

This code only handles the second and third case. Should probably handle the first case too. Need to check the value of the combo box.

Reply:
For now, I've fixed this by assuming for unknown that the person is dead ? (Since it is weird to be having a unknown living status but still be most definitely living )